### PR TITLE
metadata creation to support partograph form initialization and rende…

### PR DESCRIPTION
**Pull request by ckote**
metadata creation to support partograph form initialization and rendering

### CODE REVIEW

**Comment by ckote** [comment]
@makombe please review

File: api/src/main/java/org/openmrs/module/kenyaemr/metadata/MchMetadata.java [hunk 1]
```diff
@@ -99,12 +101,14 @@ public void install() {
...
 		install(encounterType("MCH Mother Consultation", "Collection of mother data during MCH visit", _EncounterType.MCHMS_CONSULTATION));
 		install(encounterType("MCH Mother Discontinuation", "Discontinuation of mother from MCH program", _EncounterType.MCHMS_DISCONTINUATION));
+		install(encounterType("MCH Partograph", "Labour progression monitoring form", _EncounterType.MCHMS_PARTOGRAPH));
 
 		install(form("MCH-MS Enrollment", "MCH-MS Enrollment", _EncounterType.MCHMS_ENROLLMENT, "1.0", _Form.MCHMS_ENROLLMENT));
 		install(form("MCH Antenatal Visit", "MCH antenatal visit form", _EncounterType.MCHMS_CONSULTATION, "1.0", _Form.MCHMS_ANTENATAL_VISIT));
 		install(form("MCH Postnatal Visit", "MCH postnatal visit form", _EncounterType.MCHMS_CONSULTATION, "1.0", _Form.MCHMS_POSTNATAL_VISIT));
 		install(form("Delivery", "MCH-MS delivery form", _EncounterType.MCHMS_CONSULTATION, "1.0", _Form.MCHMS_DELIVERY));
 		install(form("Discharge", "MCH-MS discharge form", _EncounterType.MCHMS_CONSULTATION, "1.0", _Form.MCHMS_DISCHARGE));
+		install(form("MCH_Partograph", "MCH-MS labour form", _EncounterType.MCHMS_PARTOGRAPH, "1.0", _Form.MCHMS_PARTOGRAPH));
```
Comment (makombe) [inline api/src/main/java/org/openmrs/module/kenyaemr/metadata/MchMetadata.java:111]:
```suggestion
		install(form("MCH Partograph", "MCH-MS labour form", _EncounterType.MCHMS_PARTOGRAPH, "1.0", _Form.MCHMS_PARTOGRAPH));
```

Comment (makombe) [reply api/src/main/java/org/openmrs/module/kenyaemr/metadata/MchMetadata.java:111]: @ckote you can remove the underscore from the name. That is the name of the form that will be visible to the user

Comment (ckote) [reply api/src/main/java/org/openmrs/module/kenyaemr/metadata/MchMetadata.java]: @makombe kindly recheck

**Pull request merged**